crud.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+
+from sqlalchemy.orm import Session
+
+import schemas
```

Comment:
The `schemas` module is imported but not used correctly in the `create_author` and `create_book_for_author` functions. Ensure that the `AuthorCreate` and `BookCreate` schemas are correctly defined and used in these functions.